docs/customer-portal/cp-documents.md

```diff
@@ -1,3 +1,5 @@
 # Documents
 
 View all **Documents** associated to your account. This could include feature lists or payment requirements.
+
+You can also view the documents by just clicking on the uploaded document under **File Name** column.
```

Comment:
You can also view the documents by clicking on the uploaded document in the File Name column.

Reply:
corrected